packages/client/components/BottomControlBarReadyButton.tsx

```diff
@@ -115,29 +105,30 @@ const BottomControlBarReadyButton = (props: Props) => {
     })
 
     setDelaySeconds(isNotEditing ? 30 : undefined)
-  }, [localPhase.phaseType === 'reflect', reflectionGroups, localPhase])
+  }, [localPhase.phaseType === 'reflect', reflectionGroups, localPhase.reflectPrompts])
 
   useEffect(() => {
     if (localPhase.phaseType !== 'group') return
+    if (reflectionGroups?.every((group) => group.reflections.length === 1)) return
 
     const isNotDragging = reflectionGroups?.every((group) =>
       group.reflections?.every(
-        (reflection) => !reflection.isViewerDragging && !reflection.isDropping
+        ({isDropping, isEditing, isViewerDragging}) =>
+          !isViewerDragging && !isDropping && !isEditing
       )
     )
 
     setDelaySeconds(isNotDragging ? 30 : undefined)
-  }, [localPhase.phaseType === 'group', reflectionGroups, localPhase])
+  }, [localPhase.phaseType === 'group', reflectionGroups])
 
   useEffect(() => {
     if (localPhase.phaseType !== 'vote') return
 
-    const teamVotesRemaining = votesRemaining || 0
-    const myVotesRemaining = viewerMeetingMember?.votesRemaining || 0
-    const isNotVoting = teamVotesRemaining === 0 || myVotesRemaining === 0
+    const teamVotesRemaining = votesRemaining ?? 0
+    const noVotesRemaining = teamVotesRemaining === 0
```

Comment:
+1 this seems a bit redundant, why not just `noVotesRemaining = !votesRemaining`? Or if you want to have it more explicit `noVotesRemaining = votesRemaining ?? 0 === 0`